tgui/packages/tgui/interfaces/HighLuminosityEyesMenu.tsx

```diff
@@ -0,0 +1,171 @@
+import { BooleanLike } from '../../common/react';
+import { useBackend } from '../backend';
+import { Button, ColorBox, Input, LabeledList, NumberInput, Section } from '../components';
+import { Window } from '../layouts';
+
+type EyeColorData = {
+  mode: BooleanLike;
+  hasOwner: BooleanLike;
+  left: string;
+  right: string;
+};
+
+type HighLuminosityEyesMenuData = {
```

Comment:
i appreciate this effort but it's a bit lengthy for a local type name

"Data" is fine